rules/Renaming/Rector/PropertyFetch/RenamePropertyRector.php

```diff
@@ -124,7 +125,6 @@ private function processFromPropertyFetch(PropertyFetch $propertyFetch): ?Proper
                 continue;
             }
 
-            $nodeVarType = $this->nodeTypeResolver->getType($propertyFetch->var);
```

Comment:
Before this PR the expensive type-resolving was only needed after the cheap name checks succeeded.

I think instead we should leave the check in the loop but do it only after the classlike instanceof check.

Do you have a concrete real world example you are optimizing for?

Reply:
I don't have, probably isset variable inside loop is better..., wdyt?

Reply:
Ok, Updated 👍